src/robusta/integrations/prometheus/models.py

```diff
@@ -109,7 +109,7 @@ def get_description(self) -> str:
             )
         return clean_description
 
-    def __get_alert_subject(self) -> FindingSubject:
+    def get_alert_subject(self) -> FindingSubject:
```

Comment:
same comment about __ prefix but in reverse. why removing it? it is a private function

Reply:
I'm now calling it from the new playbook, so it can't be private